src/zarr/api/asynchronous.py

```diff
@@ -817,19 +819,40 @@ async def create(
     shape : int or tuple of ints
         Array shape.
     chunks : int or tuple of ints, optional
-        Chunk shape. If True, will be guessed from `shape` and `dtype`. If
-        False, will be set to `shape`, i.e., single chunk for the whole array.
-        If an int, the chunk size in each dimension will be given by the value
-        of `chunks`. Default is True.
+        The shape of the array's chunks.
+        V2 only. V3 arrays should use `chunk_shape` instead.
+        Default values are guessed based on the shape and dtype.
     dtype : str or dtype, optional
         NumPy dtype.
+    chunk_shape : int or tuple of ints, optional
+        The shape of the Array's chunks (default is None).
+        V3 only. V2 arrays should use `chunks` instead.
+    chunk_key_encoding : ChunkKeyEncoding, optional
+        A specification of how the chunk keys are represented in storage.
+        V3 only. V2 arrays should use `dimension_separator` instead.
+        Default is ("default", "/").
+    codecs : Sequence of Codecs or dicts, optional
+        An iterable of Codec or dict serializations thereof. The elements of
+        this collection specify the transformation from array values to stored bytes.
+        V3 only. V2 arrays should use `filters` and `compressor` instead.
+        If no codecs are provided, default codecs will be used:
+        - For numeric arrays, the default is `BytesCodec` and `ZstdCodec`.
+        - For Unicode strings, the default is `VLenUTF8Codec`.
+        - For bytes or objects, the default is `VLenBytesCodec`.
+        These defaults can be changed using the `array.v3_default_codecs` variable in the Zarr config.
     compressor : Codec, optional
-        Primary compressor.
+        Primary compressor to compress chunk data.
+        V2 only. V3 arrays should use `codecs` instead.
+        If neither `compressor` nor `filters` are provided, a default compressor will be used:
```

Comment:
```suggestion
        If neither ``compressor`` nor ``filters`` are provided, a default compressor will be used:
```
Single ticks are sphinx links, double backticks are code (and I think these should be code instead of attempting to link somewhere)